modules/io/common/src/main/java/org/locationtech/jts/io/geojson/GeoJsonWriter.java

```diff
@@ -34,7 +34,7 @@
 
 
 /**
- * Writes {@link Geometry}s as JSON fragments in GeoJson format.
+ * Writes {@link Geometry}s, {@link Feature}s, and {@link FeatureCollection}s as JSON fragments in GeoJson format.
```

Comment:
The Feature and FeatureCollection are full JSON documents, not just fragments I think?